### DISCUSSION TITLE
Atlas Dark (Black)

### DISCUSSION THREAD

**Discussion by Zusier** [Atlas-OS/Atlas#162] [Community Artwork]
Dark themed wallpaper for my fellow night owls.. have a few variants as I don't enjoy pure black
![Atlas_Dark](https://user-images.githubusercontent.com/46111482/167278234-20daa212-3712-4dbb-b359-0c81d22974c6.png)
![Atlas_Dark_gray](https://user-images.githubusercontent.com/46111482/167278237-bcc84d5e-5f01-4bc9-883d-83a7710fcb10.png)
![Atlas_Dark_lighter_gray](https://user-images.githubusercontent.com/46111482/167278239-bf4ce836-cc27-4def-91b4-ca759eb85679.png)

Photoshop file: [Atlas_Dark_PSD.zip](https://github.com/Atlas-OS/Atlas/files/8646198/Atlas_Dark_PSD.zip)

**Comment by TechStudent10** [comment]
I absolutely love it! These should be the default backgrounds for Dark Mode.
